service-coordtransform/src/main/java/fi/nls/paikkatietoikkuna/coordtransform/CoordinateTransformationActionHandler.java

```diff
@@ -108,9 +111,13 @@ public void init() {
 
     @Override
     public void handlePost(ActionParameters params) throws ActionException {
+        String transformType = params.getHttpParam(PARAM_TRANSFORM_TYPE);
+        if ("F2R".equals(transformType)){ // parse file to array without transformation
+            readFileToJsonResonse(params);
```

Comment:
Is there a reason why this is not part of the switch statement below and return is not used so in addition the response being written the rest of the method is run too?